src/app/shared/shared.module.ts

```diff
@@ -138,6 +142,8 @@ const modules = [
     ...modules,
     CountdownTimerComponent,
     UnsupportedChainMsgComponent,
+    SafeUrlPipe,
```

Comment:
if you put them in the pipes array you won't have to include them here (the components array too, it's spreaded just 3 lines above)